src/xcode/ENA/ENA/Source/Services/HealthCertificate/Models/HealthCertificate.swift

```diff
@@ -306,9 +306,9 @@ final class HealthCertificate: Codable, Equatable, Comparable, RecycleBinIdentif
 
 	var validityStateIsConsideredNewsworthy: Bool {
 		switch validityState {
-		case .valid:
+		case .valid, .expired, .expiringSoon:
 			return false
-		case .invalid, .blocked, .revoked, .expired, .expiringSoon:
+		case .invalid, .blocked, .revoked:
```

Comment:
As discussed today in Daily: when that bug fix rise a side effect to the former mentioned fix, we'll create a new bug for that, because that bug fix here, is more important than the former one.